plugins/modules/azure_rm_adapplication.py

```diff
@@ -146,33 +146,103 @@
     optional_claims:
         description:
             - Declare the optional claims for the application.
-        type: list
-        elements: dict
+        type: dict
         suboptions:
-            name:
-                description:
-                    - The name of the optional claim.
-                type: str
-                required: True
-            source:
+            access_token:
                 description:
-                    - The source (directory object) of the claim.
-                    - There are predefined claims and user-defined claims from extension properties.
-                    - If the source value is null, the claim is a predefined optional claim.
-                    - If the source value is user, the value in the name property is the extension property from the user object.
-                type: str
-            essential:
+                    - The optional claims returned in the JWT access token
+                type: list
+                elements: dict
+                suboptions:
+                    name:
+                        description:
+                            - The name of the optional claim.
+                        type: str
+                        required: True
+                    source:
+                        description:
+                            - The source (directory object) of the claim.
+                            - There are predefined claims and user-defined claims from extension properties.
+                            - If the source value is null, the claim is a predefined optional claim.
+                            - If the source value is user, the value in the name property is the extension property from the user object.
+                        type: str
+                    essential:
+                        description:
+                            - If the value is true, the claim specified by the client is necessary to ensure a 
```

Comment:
This line feed is not supported by the document format. It is recommended that you modify all such line breaks. Thank you!

Reply:
Apologies for the back-and-forth here, I took some time to sit down and learn how to use the yaml linter and sanity checker properly today. These should now all be fixed.